packages/flutter/lib/src/services/asset_bundle.dart

```diff
@@ -62,8 +62,12 @@ abstract class AssetBundle {
   /// If the `cache` argument is set to false, then the data will not be
   /// cached, and reading the data may bypass the cache. This is useful if the
   /// caller is going to be doing its own caching. (It might not be cached if
-  /// it's set to true either, that depends on the asset bundle
-  /// implementation.)
+  /// it's set to true either, depending on the asset bundle implementation.)
+  ///
+  /// The function expects the stored string to be UTF-8-encoded.
+  /// [Utf8Codec] will be used for decoding the string and the decoding process
+  /// is delegated to an isolate for strings larger than 50 KB to avoid jank on
+  /// the main thread.
```

Comment:
I re-read this a couple times to wrap my head around it, this might be a little clearer, WDYT?

```suggestion
  /// The function expects the stored string to be UTF-8-encoded as
  /// [Utf8Codec] will be used for decoding the string. If the string is
  /// larger than 50 KB, the decoding process is delegated to an
  /// isolate to avoid jank on the main thread.
```

Reply:
Thanks!